src/lib/services/verification/move/index.ts

```diff
@@ -84,17 +84,43 @@ export const useMoveVerifyInfo = (
       return getMoveVerifyInfo(address, moduleName);
     },
     {
-      enabled: layer === "1" && Boolean(address && moduleName),
+      enabled: layer === "1",
```

Comment:
- check `undefined` for both address and name
- throw instead of `null`
- remove `Nullable`